src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  keyPress: string,
```

Comment:
I think that `keyPressed` better describes the intention of this variable
```suggestion
  keyPressed: string,
```